documentation/book/proc-configuring-kafka-mirror-maker-abort-on-send-failure.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-abort-on-send-failure.adoc
+
+[id='proc-configuring-kafka-mirror-maker-abort-on-send-failure-{context}']
+= Configuring the Mirror Maker to exit on a failed send
+
+Configure whether the Mirror Maker should exit on a failed send using the `KafkaMirrorMaker.spec.producer.abortOnSendFailure` property.
```

Comment:
What I was saying is that because exiting is the default behavior I would make this chapter in the opposite way, so talking about how to not exit but moving to the next message if one fails (changing title, description and reverting back to `false`). Wdyt @scholzj ?

Reply:
Yeah, if exit is the default it should be probably written the other way around.

Reply:
Suggested wording:

By default, if an error occurs when sending a message from Kafka Mirror Maker to a Kafka cluster:

* The Kafka Mirror Maker container is terminated in {ProductPlatformName}. 

* The container is then recreated.

To ignore sending errors and move to the next message, you can set the `KafkaMirrorMaker.spec.producer.abortOnSendFailure` property to `false`.